### ISSUE TITLE
TabNet. Unable to reproduce the 99% accuracy on Poker Hand dataset.

### ISSUE THREAD

**Issue by mgrankin** [google-research/google-research#191]
We're trying to reproduce the results from a TabNet paper. There is a remarkable 99% accuracy on Poker Hand dataset. We can't reproduce the result, we've got only 54%. Can you share the code, so we can check what's wrong with our setup?

**Comment by muellerzr** [comment]
To add to this, we understand that there’s a lot of hyperparameter tuning to be done. We also tried following both the ones suggested in the paper and our own hyper parameter search which only achieved ~62% accuracy.

**Comment by mgrankin** [comment]
https://github.com/fmobrj/pokerhand_tabular/blob/master/poker_hand_induction_fastai_x_tabnet_final.ipynb

**Comment by cwerner** [comment]
Hi there.

I figure this problem was resolved? Could you briefly explain if there was a bug in the PyTorch implementation of the code...? I'd be interested in using the PyTorch model and want to be sure that there's no implementation bug. Thanks

**Comment by muellerzr** [comment]
The issue was how the data was being prepared. Everything was treated as categorical (the notebook above shows).

**Comment by cwerner** [comment]
Thanks @muellerzr ! Will jump right in...

And thanks for your work and contributions in the fast.ai community. Any intuition why the fast.ai models beats? Not that I am complaining ;-)

**Comment by muellerzr** [comment]
We’re looking into it but a study wasn’t really ever done to compare how fastai does on baselines (doing so **now**) and so plenty to be discovered :)

**Comment by Optimox** [comment]
@cwerner we released a bugfix yesterday on pytorch-tabnet, so everything should be working fine now!

**Comment by cwerner** [comment]
@Optimox great - thanks for the heads-up
